DatWeatherDoe/UI/Menu Bar/StatusItemManager.swift

```diff
@@ -20,6 +20,18 @@ final class StatusItemManager {
     init(menu: NSMenu, configureSelector: Selector) {
         statusItem.menu = menu
         statusItem.button?.action = configureSelector
+
+        // Category icons in dropdown menu
+        locationMenuItem?.image = WeatherConditionImageMapper().map(.location)
```

Comment:
As mentioned above, I think these images should live in their own mapper outside of `WeatherCondition`. Separately, whatever that class is called, you can instantiate it once and call `map` repeatedly on the same instance.

Reply:
Okay. I'll check it out.